### PR TITLE
Move dashboard embed to use powerbi javascript nuget.

### PR DESCRIPTION
**Pull request by ali-hamud**
## Description
1) Move dashboard embed to use powerbi javascript nuget.
2) style

## Questison
please answer the following questions. put x inside [ ] (e.g. [x])

### What inside?
- [ ] Bug Fixes?
- [ ] New Features?
- [ ] Documentation?
--- refactoring.

### Is pull request totally generated from swagger file?
- [ ] Yes.
- [ ] No, part of it is auto-generated.

### Backward compatibility break?
- [ ] Yes. Pull request breaks backward compatibility!

[Learn more about backward compatibility.](BackwardCompatibility.md)

### CODE REVIEW

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master [hunk 1]
```diff
@@ -58,42 +37,42 @@
...
-
             // get the access token.
             accessToken = document.getElementById('MainContent_accessTokenTextbox').value;
 
-            // return if no a
-            if ("" === accessToken)
-                return;
+            // Embed configuration used to describe the what and how to embed.
+            // This object is used when calling powerbi.embed.
+            // This also includes settings and options such as filters.
```
Comment (ArinaHantsis) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: No filters for dashboards ... Please update comment

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master [hunk 2]
```diff
@@ -58,42 +37,42 @@
...
 
-            // push the message.
-            iframe = document.getElementById('iFrameEmbedDashboard');
-            iframe.contentWindow.postMessage(message, "*");
-        }
+            // Embed the dashboard and display it within the div container.
+            var dashboard = powerbi.embed(dashboardContainer, config);
+
+            // dashboard.on will add an event handler which prints to Log window.
+            dashboard.on("tileClicked", function (event) {
```
Comment (ArinaHantsis) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: Thanks!

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master [hunk 3]
```diff
@@ -58,42 +37,42 @@
...
+                var logView = document.getElementById('logView');
+                logView.innerHTML = logView.innerHTML + "Tile Clicked<br/>";
+                alert(event);
```
Comment (ArinaHantsis) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: The first alert is probably for debugging .....

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master [hunk 4]
```diff
@@ -58,42 +37,42 @@
...
-        // post the auth token to the iFrame. 
-        function postActionLoadDashboard() {
-
             // get the access token.
             accessToken = document.getElementById('MainContent_accessTokenTextbox').value;
 
-            // return if no a
-            if ("" === accessToken)
-                return;
+            // Embed configuration used to describe the what and how to embed.
```
Comment (noanu) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: I would have removed this sentence, or at least rephrase it..

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master (hunk 1 above)
Comment (noanu) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: filters aren't relevant for dashboards

File: samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master (hunk 4 above)
Comment (ali-hamud) [inline samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Site.Master]: nooo 👯 it's good

**Pull request merged**